vms/platformvm/txs/executor/helpers_test.go

```diff
@@ -352,68 +350,83 @@ func defaultFx(clk *mockable.Clock, log logging.Logger, isBootstrapped bool) fx.
 	return res
 }
 
-func buildGenesisTest(ctx *snow.Context) []byte {
-	genesisUTXOs := make([]api.UTXO, len(preFundedKeys))
+func buildGenesisTest(ctx *snow.Context) *genesis.Genesis {
+	genesisUtxos := make([]*genesis.UTXO, len(preFundedKeys))
```

Comment:
these lines are basically what `platformvmSS.BuildGenesis` used to do. Pulled up here and in other helpers_test.go files. In a following PR a drop the code duplication and centralize genesis creation in UTs